k9mail-library/src/test/java/com/fsck/k9/mail/store/webdav/WebDavStoreTest.java

```diff
@@ -120,6 +123,34 @@ public void createUri_withSettingsWithExtras_shouldProvideUriWithExtras() {
         assertEquals("webdav://user:password@example.org:123456/path%7CauthPath%7CmailboxPath", result);
     }
 
+    @Test
+    public void can_create_uris_from_server_settings() {
+
+        assertEquals("webdav://username:password@example.org:123456/%7C%7C",
```

Comment:
I find this very painful to read. Again, please use the arrange-act-assert pattern.